### ISSUE TITLE
PSA repo split - Q2 summary issue

### ISSUE THREAD

**Issue by daverodgman** [Mbed-TLS/mbedtls#7221]

**Comment by daverodgman** [comment]
** Epic checklist **

- Is epic fully broken down?
  - No, agreed initial set of issues but epic/issue creation TBD
- Epic estimate
  - TBD
- DoD
  - Complete when the two deliverables above are complete and at the end of Q2
- Experts list
  - @ronald-cron-arm, @adeaarm, Julian Hall
- Design documentation
  -  Present, in the initial PR. TODO - add links
- Dependencies
  - Depend on prompt feedback - @laumor01 may be able to help manage this dependency?
  - Depend on #7059 and similar follow-ups - not must-have or urgent but important long-term
  - CI depends on support from @bensze01 
- Resources needed
  - n/a
- Infra/Test automation/CI needs
  - Nothing at the moment - maybe something minor will come out of the CI work?
- Requirement link
  - https://jira.arm.com/browse/SECLIB-622
- Risks
  - Feedback will highlight some major unforeseen tasks
  - Code size may be unacceptable (doesn't affect delivering this q)
  - Reliance on prompt feedback
- Testing notes
  - Need to document how to validate the updates
  - Based on running the PSA test suites (from all.sh)
  - Want to add the PSA Arch tests (issue exists for Q1)

**Comment by gilles-peskine-arm** [comment]
The project is done (and we haven't been working from this summary issue for a while anyway) so this issue is no longer relevant.
